src/net_processing.cpp

```diff
@@ -3230,12 +3231,21 @@ bool SendMessages(CNode* pto, CConnman& connman, const std::atomic<bool>& interr
                         break;
                     }
                     pBestIndex = pindex;
+                    bool isPrevDevnetGenesisBlock = false;
+                    if (params.NetworkIDString() == CBaseChainParams::DEVNET &&
+                        pindex->pprev != nullptr &&
+                        pindex->pprev->GetBlockHash() == params.DevNetGenesisBlock().GetHash()) {
```

Comment:
Let's align with the way we check for devnet genesis block in other parts of the code:

```
if (!consensusParams.hashDevnetGenesisBlock.IsNull() &&
    pindex->pprev != nullptr &&
    pindex->pprev->GetBlockHash() == consensusParams.hashDevnetGenesisBlock) {
```

Reply:
Done (squashed into original commit as this is a small PR)